apt/private/deb_translate_lock.bzl

```diff
@@ -78,8 +162,29 @@ def _deb_translate_lock_impl(rctx):
             ),
         )
 
+    # TODO: rework lockfile to include architecure information and merge these two loops
```

Comment:
That's exactly what I did in https://github.com/jjmaestro/rules_distroless/blob/b0f06d770cb6702179aa7fc994d1d1541cb4e8f7/apt/private/deb_translate_lock.bzl#L25-L40. Also, a big part of the cleanup in this file comes from another refactor, #96.